third-party/fedora31/kernel/BUCK

```diff
@@ -0,0 +1,49 @@
+remote_file(
+    name = "kernel-core-5.3.7-301.fc31.x86_64.rpm",
+    url = "http://mirrors.kernel.org/fedora/releases/31/Everything/x86_64/os/Packages/k/kernel-core-5.3.7-301.fc31.x86_64.rpm",
+    sha1 = "dae4263b23930c4f25a7eae28a1fda606abfd561",
+    out = "kernel-core-5.3.7-301.fc31.x86_64.rpm",
+)
+
+remote_file(
+    name = "kernel-headers-5.3.6-300.fc31.x86_64.rpm",
+    url = "http://mirrors.kernel.org/fedora/releases/31/Everything/x86_64/os/Packages/k/kernel-headers-5.3.6-300.fc31.x86_64.rpm",
+    sha1 = "7fe40f1592bb8ac3c47d3f97b447a7bf6f2e0a05",
+    out = "kernel-headers-5.3.6-300.fc31.x86_64.rpm",
+)
+
+remote_file(
+    name = "kernel-devel-5.3.7-301.fc31.x86_64.rpm", 
+    url = "http://mirrors.kernel.org/fedora/releases/31/Everything/x86_64/os/Packages/k/kernel-devel-5.3.7-301.fc31.x86_64.rpm",
+    sha1 = "ff5bebf0be74228591bc046175815a3e5ce927f2",
+    out = "kernel-devel-5.3.7-301.fc31.x86_64.rpm",
+)
+
+genrule(
+    name = "5.3.7-301.fc31.x86_64-rpm-exploded",
+    out = ".",
+    cmd = """
+        cd $OUT
+        rpm2cpio $(location :kernel-core-5.3.7-301.fc31.x86_64.rpm) | cpio -idm
+        rm -r lib/modules/5.3.7-301.fc31.x86_64/build
```

Comment:
can you add a comment here that this is because they are symlinks that will not exist on the host, so buck complains?

Reply:
We could potentially get the rpm out of the snapshot, that would basically just be a different URL for the `remote_file` target.  The same thing applies to all of the `fedora31` third-party libraries, but we can solve that problem later.

Reply:
Also, fwiw, getting this out of the snapshot would still require network requests since the snapshot content isn't local.